src/types/supabase.ts

```diff
@@ -69,6 +83,20 @@ export interface Extension {
   status: string | null;
 }
 
+export interface Proposal {
```

Comment:
This type needs an update on frontend and backend, will have more updates in a separate issue.